docs/metrics.md

```diff
@@ -126,8 +126,8 @@ scp.timeout.nominate                     | meter     | timeouts in nomination
 scp.timeout.prepare                      | meter     | timeouts in ballot protocol
 scp.timing.nominated                     | timer     | time spent in nomination
 scp.timing.externalized                  | timer     | time spent in ballot protocol
-scp.timing.externalize-lag               | timer     | delay between first externalize message and local node externalizing
-scp.timing.externalize-delay             | timer     | delay between local node externalizing and later externalize messages
+scp.timing.self-to-first-externalize-lag | timer     | delay between first externalize message and local node externalizing
```

Comment:
`self-to-` sounds like self is *before*. So maybe this should be `self-from-first-externalize-lag` ?

Reply:
actually using the "from" variant is bad, we should just use "to" so in this case `first-to-self-externalize-lag`